lib/seed/production_experiment_test_seeder.rb

```diff
@@ -0,0 +1,28 @@
+module Seed
+  class ProductionExperimentTestSeeder
+    def self.call(user_id:, days_till_appointment: 1)
+      ExperimentSeeder.call(experiment_name: "production test", experiment_type: "current_patients")
+
+      now = Time.current
+      today = Date.current
+      user = User.find(user_id)
+      facility = user.registration_facility
+
+      patient = Patient.create!(id: SecureRandom.uuid, full_name: "Hari AB Tester", status: "active", device_created_at: now, device_updated_at: now, registration_user: user)
+      patient.phone_numbers.create!(id: SecureRandom.uuid, number: "8675309", phone_type: "mobile", active: true, device_created_at: now, device_updated_at: now)
```

Comment:
constant for the fake number? In fact, we could probably just have an array of the `full_names` and loop thru them, it looks like everything is the same here...and then Nilenso can do the phone number updates manually afterwards.

Reply:
That's also a valid point. I'll make that change.